tests/baselines/reference/user/webpack.log

```diff
@@ -4,6 +4,11 @@ lib/ExternalModule.js(305,28): error TS2554: Expected 1 arguments, but got 0.
 lib/Stats.js(68,31): error TS2554: Expected 3 arguments, but got 2.
 lib/library/SystemLibraryPlugin.js(127,35): error TS2554: Expected 1 arguments, but got 0.
 lib/optimize/ConcatenatedModule.js(205,32): error TS2554: Expected 1 arguments, but got 0.
+lib/wasm-async/AsyncWebAssemblyModulesPlugin.js(36,15): error TS2304: Cannot find name 'Chunk'.
```

Comment:
Previously, these were in a bad state; values with these types were able to resolve properties, but actual type references failed to resolve -- but without an error.

Now things are the same, except that there is an error.

This is not that bad of a regression, although I haven't figured out how to work around the underlying problem.

Reply:
Update: The error does get issued if you use the property. I think this new error is a result of the PR that adds checking to `@property` tags.